db/db.pl

```diff
@@ -2089,6 +2100,14 @@ sub parseArgs {
 parseArgs(3) if ($ARGV[1] =~ /^(restore)$/);
 
 $main::userAgent = LWP::UserAgent->new(timeout => $ESTIMEOUT + 5, keep_alive => 5);
+if ($CLIENTCERT ne "") {
+    $main::userAgent->ssl_opts(
+        SSL_verify_mode => $SECURE,
+        verify_hostname=> $SECURE,
+        SSL_cert_file => $CLIENTCERT,
+        SSL_key_file => $CLIENTKEY
+    )
+}
```

Comment:
maybe add an else here that if $SECURE is set to 0 but CLIENTCERT isn't set it still turns off verify?

Reply:
yes, makes sense